content/en/tracing/trace_collection/library_config/php.md

```diff
@@ -356,6 +356,14 @@ Propagation styles to use when extracting tracing headers. If using multiple sty
   - [B3 single header][8]
   - Datadog
 
+`DD_DBM_PROPAGATION_MODE`
+: **INI**: `datadog.dbm_propagation_mode`<br>
+**Default**: `'disabled'`<br>
+: Enables linking between data sent from APM and the Database Monitoring product when set to `'service'` or `'full'`.<br>
```

Comment:
If this was intentional feel free to leave it. But if you want this to be consistent with formatting of other sections, you can remove the colon
![image](https://user-images.githubusercontent.com/41168354/227595341-7efd91a3-eec6-4a03-895d-2d49615524ae.png)

```suggestion
Enables linking between data sent from APM and the Database Monitoring product when set to `'service'` or `'full'`.<br>
```

Reply:
Thanks. Approved